perf/benchmarkapps/GrpcClient/GrpcClient.csproj

```diff
@@ -16,6 +16,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
 
+    <PackageReference Include="Microsoft.Crank.EventSources" Version="0.2.0-alpha.21255.1" />
```

Comment:
Move the version to this file - https://github.com/grpc/grpc-dotnet/blob/master/build/dependencies.props - and reference it like the other `PackageReference` elements.